Wrappers/Python/cil/optimisation/functions/IndicatorBox.py

```diff
@@ -34,83 +38,336 @@ class IndicatorBox(Function):
     
     '''
     
-    def __init__(self,lower=-np.inf,upper=np.inf):
+    def __init__(self, lower=None, upper=None):
         '''creator
 
-        Parameters:
-        -----------
-        lower : float, DataContainer or numpy array, default ``-np.inf``
-          Lower bound
-        upper : float, DataContainer or numpy array, default ``np.inf``
-          upper bound
+        Parameters
+        ----------
+        
+            lower : float, DataContainer or numpy array, default None
+                Lower bound. If set to None, it is equivalent to ``-np.inf``.
+            upper : float, DataContainer or numpy array, default None
+                Upper bound. If set to None, it is equivalent to ``np.inf``.
         
-        If passed a DataContainer or numpy array, the bounds can be set to different values for each element.
+        If passed a ``DataContainer`` or ``numpy array``, the bounds can be set to different values for each element.
```

Comment:
```suggestion
        If passed a ``DataContainer`` (for example ``ImageData`` or ``AcquisitionData``) or ``numpy array``, the bounds can be set to different values for each element.
```
I find this clearer since not all users will be aware that ID and AD are special types of DataContainer

Reply:
updated